### ISSUE TITLE
shift+Tab key combination not working in the Select Location and Language menus

### ISSUE THREAD

**Issue by madalincm** [mozilla-mobile/mozilla-vpn-client#716]
#### Affected versions:
- v. 2.0.5 (202103181536); 

#### Affected Platforms:
- Windows 10;

#### Prerequisites:
- Have a FxA account with active subscription for Mozilla VPN;

#### Steps to reproduce:
1. Open the VPN;
2. Open the Select Location menu;
3. Use the Tab key to navigate inside the menu;
4. Use the shift+Tab key combination to navigate backwards in the menu ;

#### Expected result:
- The shift+Tab key combination lets the user navigate through each countries listed in the menu;

#### Actual result:
- When using shift+Tab key combination the focus jups directly to the back arrou and the user is unable to navigate;

#### Notes:
- The issue is also reproducing in the Language menu

**Comment by lesleyjanenorton** [comment]
Hey! I'm going to close this as this functionality is intentional - the feedback from the accesibility team was that these "backtabbing" while in these lists should take the user to the "back" button", while the "up-arrow" should move the user to the previous language/server country, etc.

**Comment by madalincm** [comment]
Hello, thanks for your input. 
The problem here is that after using the key combination shift + tab and user is sent to the back button, the tab key is not working properly. The user can't navigate properly between all the options.
Try the following steps:
1. Enter the Select Language menu
2. Press the tab key a few times
3. Use shift + tab key combination 
4. Press the tab key again
--- After the user uses the shift tab the next tab key presses will not navigate the user on the previous languages only from the language he was on while he first pressed shift + tab
![backtabingF](https://user-images.githubusercontent.com/15685441/111996102-59ff8900-8b22-11eb-871c-6b33dc45d4d3.gif)

**Comment by lesleyjanenorton** [comment]
If I'm understanding your comment correctly, this is as-designed. When you are focused on a list item and click shift + tab, the focus should jump to the back button. Then, if you click tab again, the focus should jump back to the last focused list item. The list is navigable by clicking the up and down arrows.

**Comment by rbillings** [comment]
This is the implementation requested by accessibility. This requires a product decision. Waiting for feedback from UX.

**Comment by ValentinaPC** [comment]
@lesleyjanenorton this was closed because it suppose to be fixed or because is working as it suppose to? Thanks!
